p2panda-rs/src/atomic/entry_signed.rs

```diff
@@ -1,14 +1,12 @@
 use std::convert::{TryFrom, TryInto};
 
 use arrayvec::ArrayVec;
-use bamboo_rs_core::entry::MAX_ENTRY_SIZE;
-use bamboo_rs_core::{Entry as BambooEntry, Signature as BambooSignature};
-use ed25519_dalek::{PublicKey, Signature};
+use bamboo_rs_core::Entry;
```

Comment:
I would keep the `BambooEntry` renaming just for readability / not confusing it with out own `atomic::Entry` struct.